io.anytype.anytype.metainfo.xml

```diff
@@ -25,7 +25,7 @@
   <launchable type="desktop-id">io.anytype.anytype.desktop</launchable>
 
   <releases>
-    <release version="0.43.1" date="2024-10-24" />
+    <release version="0.43.4" date="2024-10-31" />
```

Comment:
The new release should have been added as a new record, not replacing the last one.